operations/resiliency/Resiliency_Testing_Procedure.md

```diff
@@ -1,240 +1,386 @@
-   ```
+   If the target node for shutdown is a worker NCN, force the UAI to be created on target worker NCN with the following steps:
+   
+   1. Create labels to ensure that UAI pods will only be able to start-up on the target worker NCN.
+      
+      In this example, `ncn-w002` is the target node for shutdown on a system with only three NCN worker nodes.
 
-   ```bash
-   ncn# kubectl get nodes -l uas=False
-   NAME       STATUS   ROLES    AGE     VERSION
-   ncn-w001   Ready    <none>   4d19h   v1.18.2
-   ncn-w003   Ready    <none>   4d19h   v1.18.2
-   ```
-   It is **EXTREMELY IMPORTANT** to note that after a UAI has been created, that we clear out these labels or else when kubernetes terminates the running UAI on the NCN being shut down, it will not be able to reschedule the UAI on another pod.
+      ```bash
+      ncn# kubectl label node ncn-w001 uas=False --overwrite
+      ncn# kubectl label node ncn-w003 uas=False --overwrite
+      ```
+
+      If successful, output similar to `node/ncn-w00X labeled` will be returned.
+
+   1. Verify the labels were applied successfully.
+
+      ```bash
+      ncn# kubectl get nodes -l uas=False
+      ```
+
+      Example output:
+
+      ```
+      NAME       STATUS   ROLES    AGE     VERSION
+      ncn-w001   Ready    <none>   4d19h   v1.18.2
+      ncn-w003   Ready    <none>   4d19h   v1.18.2
+      ```
    
-   To create a UAI, see the `Validate UAI Creation` section of [Validate CSM Health](../validate_csm_health.md) and/or [Create a UAI](../UAS_user_and_admin_topics/Create_a_UAI.md).
-   To remove the labels set above after the UAI has been created, run the following:
+   1. Remove the labels set above after the UAI has been created.
+
+      > **IMPORTANT:** After a UAI has been created, labels must be cleared or else when Kubernetes terminates the running UAI on the NCN being shut down, it will not be able to reschedule the UAI on another pod.
+      
+      ```bash
+      ncn# kubectl label node ncn-w001 uas-
+      ncn# kubectl label node ncn-w003 uas-
+      ```
+
+1. Verify that WLM is configured with the appropriate workload manager within the created UAI.
    
-   ```bash
-   ncn# kubectl label node ncn-w001 uas-
-   ncn# kubectl label node ncn-w003 uas-
-   ```
-* Within the created UAI, verify that WLM is configured with the appropriate workload manager.
-  Verify the connection string of the created UAI:
+   1. Verify the connection string of the created UAI.
    
-   ```bash
-   ncn# cray uas list
-   ```
+      ```bash
+      ncn# cray uas list
+      ```
 
-   Example output:
+      Example output:
 
-   ```
-   [[results]]
-   username =  "uastest"
-   uai_host =  "ncn-w001"
-   uai_status =  "Running: Ready"
-   uai_connect_string = "ssh uastest@172.30.48.49 -p 31137 -i ~/.ssh/id_rsa"
-   uai_img =  "bis.local:5000/cray/cray-uas-sles15sp1-slurm:latest"
-   uai_age =  "1m"
-   uai_name =  "uai-uastest-5653e9b9"
-   ```
-   Login to the created UAI (example):
+      ```
```

Comment:
The output shown in this file is from a 1.3 system. Use the following for 1.5:
(You may want to replace the "vers" user name with something else?)

[[results]]
uai_age = "2m"
uai_connect_string = "ssh vers@10.103.8.170"
uai_host = "ncn-w002"
uai_img = "registry.local/cray/cray-uai-cos-2.1.70:latest"
uai_ip = "10.103.8.170"
uai_msg = ""
uai_name = "uai-vers-f8fa541f"
uai_status = "Running: Ready"
username = "vers"

Reply:
fixed